tests/unit/challenges/test_aws_utils.py

```diff
@@ -0,0 +1,199 @@
+import mock
+import boto3
```

Comment:
@nikochiko can you reorder the imports lexicographically

Reply:
Done :+1: